views/index.ejs

```diff
@@ -40,6 +47,9 @@
               <button
                 data-id="<%=item.id%>"
                 class="addItem fa-solid fa-plus"
+                <% if (item.onList) { %>
```

Comment:
can you use ternary operator for this?

Reply:
I change the disabled to boolean.. i dont remember why i wrote it with an assignment.